---
permalink: /
title: "News and Quick Preview"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

Latest Published Work [A multi-view graph contrastive learning framework for deciphering spatially resolved transcriptomics data](https://tju-zl.github.io/publication/2024-05-27-MuCoST), *Briefings in Bioinformatics*, May 27, 2024.

![Overview of MuCoST](/images/mucost_framework.png)

**Abstract**: Spatially resolved transcriptomics data are being used in a revolutionary way to decipher the spatial pattern of gene expression and the spatial architecture of cell types. Much work has been done to exploit the genomic spatial architectures of cells. Such work is based on the common assumption that gene expression profiles of spatially adjacent spots are more similar than those of more distant spots. However, related work might not consider the nonlocal spatial co-expression dependency, which can better characterize the tissue architectures. Therefore, we propose MuCoST, a Multi-view graph Contrastive learning framework for deciphering complex Spatially resolved Transcriptomic architectures with dual scale structural dependency. To achieve this, we employ spot dependency augmentation by fusing gene expression correlation and spatial location proximity, thereby enabling MuCoST to model both nonlocal spatial co-expression dependency and spatially adjacent dependency. We benchmark MuCoST on four datasets, and we compare it with other state-of-the-art spatial domain identification methods. We demonstrate that MuCoST achieves the highest accuracy on spatial domain identification from various datasets. In particular, MuCoST accurately deciphers subtle biological textures and elaborates the variation of spatially functional patterns.

Current work
---
Spatial multiomics integration (Coming soon...).

Future work
---
Spatial pseudo-time development flow (Coming soon...).

Research Interests
---
My research focuses on the analysis methods for spatially resolved omics data.

**Spatial domain identification**
I am particularly interested in borrowing and adapting innovative methods from fields such as machine learning, statistics, and computational biology (Cross-Disciplinary Techniques). My goal is to create novel algorithms and frameworks that surpass the capabilities of existing biological methods.

**Spatial Multi-Omics Data Integration**
My research focuses on the development and application of advanced methodologies for handling spatial multi-omics data in biological genomics. By leveraging cutting-edge techniques from various disciplines, I aim to enhance the performance and insights that can be derived from multi-omics data.

**Biological Genomics**
In the realm of biological genomics, my work involves exploring the spatial organization and interactions of various omics layers, including genomics, transcriptomics, proteomics, and metabolomics. Understanding these complex interactions is crucial for uncovering the underlying mechanisms of various biological processes and diseases.

**Performance Optimization**
A significant aspect of my research is focused on performance optimization. By integrating techniques from other scientific areas, I strive to achieve higher accuracy, efficiency, and robustness in the analysis and interpretation of multi-omics data.

**Collaborative Research**
I am also committed to collaborative research efforts, working alongside experts from diverse fields to push the boundaries of what can be achieved in spatial multi-omics. Interdisciplinary collaboration is key to developing innovative solutions and advancing our understanding of complex biological systems.

Superviser & Co-author
---
- **Superviser**: Shu Liang, Tongji University.
- **Co-Superviser**: Lin Wan, AMSS, Chinese Academy of Sciences.
